app/src/main/java/com/smlnskgmail/jaman/hashchecker/components/settings/impl/SharedPreferencesSettings.java

```diff
@@ -243,7 +243,7 @@ private int getIntPreference(@NonNull String key) {
         return sharedPreferences.getInt(key, 0);
     }
 
-    public static abstract class SharedPreferencesSettingsKeyExtractor {
```

Comment:
What the key point of this change? Linter issue or something else?

Reply:
Yes, checkstyle requested this change.

Reply:
You have custom Checkstyle rules? Project doesn't integrated with Checkstyle at all.

Reply:
I think it is caused by this https://github.com/hash-checker/hash-checker/blob/45d804a39f74d2809159d2e51579ba8dd695037f/app/checkstyle.gradle#L3